test/check-application

```diff
@@ -1569,6 +1569,95 @@ class TestFiles(testlib.MachineCase):
         b.click("li[data-location='/home/admin'] a")
         self.assert_last_breadcrumb("admin")
 
+        # Edit multiple files
+        create_multiple_dirs_sh = """
+            mkdir /home/admin/multiple
+            mkdir /home/admin/multiple/testdir
+        """
+        for i in range(20):
+            create_multiple_dirs_sh += f"touch /home/admin/multiple/filename{i}\n"
+
+        self.write_file("/usr/local/bin/create-multiple-dirs.sh", create_multiple_dirs_sh, perm="755")
+        m.execute("runuser -u admin /usr/local/bin/create-multiple-dirs.sh")
+        b.mouse("[data-item='multiple']", "dblclick")
+        b.wait_not_present(".pf-v5-c-empty-state")
+        self.assert_last_breadcrumb("multiple")
+
+        # Editing two files
+        b.click("[data-item='filename1']")
+        b.mouse("[data-item='filename2']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "contextmenu")
+        b.click("button:contains('Edit permissions')")
+
+        # TODO: test for title contains file1,file2
+        b.wait_in_text("#edit-permissions-owner", "admin")
+        b.wait_in_text("#edit-permissions-group", "admin")
+        b.select_from_dropdown("#edit-permissions-owner-access", "read-only")
+        b.select_from_dropdown("#edit-permissions-group-access", "no-access")
+        b.select_from_dropdown("#edit-permissions-other-access", "no-access")
+        b.click(".pf-v5-c-modal-box button.pf-m-primary")
+        b.wait_not_present(".pf-v5-c-modal-box")
+        self.assertEqual(self.stat("%A", "/home/admin/multiple/filename1"), "-r--------")
+        self.assertEqual(self.stat("%A", "/home/admin/multiple/filename2"), "-r--------")
+
+        # No owner/group shown if ownership differs
+        m.execute("chown root: /home/admin/multiple/filename1")
+        b.wait_in_text("[data-item='filename1'] .item-owner", "root")
+
+        # HACK: the selected state contains a copy of files state so permissions information is not updated.
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "click", ctrlKey=True)
+        b.mouse("[data-item='filename1']", "contextmenu")
+        b.click("button:contains('Edit permissions')")
+
+        b.wait_visible("#edit-permissions-owner-access")
+        b.wait_not_present("#edit-permissions-owner")
+        b.wait_not_present("#edit-permissions-group")
```

Comment:
... but the rwx parts should be visible, and after editing they should all be the same. Please cover this case as well.